pineappl_capi/src/lib.rs

```diff
@@ -895,6 +895,42 @@ pub unsafe extern "C" fn pineappl_grid_split_lumi(grid: *mut Grid) {
     grid.split_channels();
 }
 
+/// Change the particle ID representation of a given Grid.
+///
+/// # Safety
+///
+/// If `grid` does not point to a valid `Grid` object (for example when `grid` is the null pointer)
+/// or if `pid_basis` doe not refer to a correct basis, then this function is not safe to call.
+#[no_mangle]
+pub unsafe extern "C" fn pineappl_rotate_pid_basis(grid: *mut Grid, pid_basis: PidBasis) {
```

Comment:
I don't think it documented it anywhere, but the naming scheme for the CAPI is: `pineappl_{struct-name}_{method-name}`. So this function should be `pineappl_grid_rotate_pid_basis`.